.github/workflows/format.yml

```diff
@@ -11,8 +11,10 @@ jobs:
     runs-on: macOS-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Tap
+        run: brew tap pointfreeco/formulae
```

Comment:
I ended up forking (already using this on swift-composable-architecture after the last PR failed to run properly when merged into `main`).